src/test/resources/application-tracing-test.properties

```diff
@@ -1,3 +1,4 @@
 terra.common.tracing.samplingRate: 1.0
 # Empty list of urlPatterns to allow all URLs to be traced.
 terra.common.tracing.urlPatterns:
+terra.common.tracking.stackdriverExportEnabled: false
```

Comment:
typo: 'tracking' instead of 'tracing'. Hopefully works after that...

Reply:
Thank  you!